ethash/src/compute.rs

```diff
@@ -528,98 +528,112 @@ fn to_hex(bytes: &[u8]) -> String {
 	unsafe { String::from_utf8_unchecked(v) }
 }
 
-#[test]
-fn test_get_cache_size() {
-	// https://github.com/ethereum/wiki/wiki/Ethash/ef6b93f9596746a088ea95d01ca2778be43ae68f#data-sizes
-	assert_eq!(16776896usize, get_cache_size(0));
-	assert_eq!(16776896usize, get_cache_size(1));
-	assert_eq!(16776896usize, get_cache_size(ETHASH_EPOCH_LENGTH - 1));
-	assert_eq!(16907456usize, get_cache_size(ETHASH_EPOCH_LENGTH));
-	assert_eq!(16907456usize, get_cache_size(ETHASH_EPOCH_LENGTH + 1));
-	assert_eq!(284950208usize, get_cache_size(2046 * ETHASH_EPOCH_LENGTH));
-	assert_eq!(285081536usize, get_cache_size(2047 * ETHASH_EPOCH_LENGTH));
-	assert_eq!(285081536usize, get_cache_size(2048 * ETHASH_EPOCH_LENGTH - 1));
-}
+#[cfg(test)]
+mod test {
+	extern crate tempdir;
```

Comment:
yep, i was leaning that direction as well. although as Rust will (in a few versions) infer `extern crate` declarations, I thought it might be best to leave them as easy to hunt down as possible